src/types.ts

```diff
@@ -165,7 +165,8 @@ export interface AppBridgeAction
   extends Action,
     DisableableAction,
     DestructableAction,
-    AppBridgeActionTarget {}
+    AppBridgeActionTarget,
+    LoadableAction {}
```

Comment:
I would love to know if this is all I have to do. I couldn't find anything else related to the AppBridgeAction type.

Reply:
I don't think `loading` is supported in App Bridge modals. It's only supported in the Contextual Save Bar. We would need to make the change to both `app-bridge` and `app-bridge-host` before doing it here. The host package is the one that actually renders the Modal: https://github.com/Shopify/app-bridge/blob/96edcdc3f8ed3d912dba5fd37e4589d72682392c/packages/app-bridge-host/src/components/Modal.tsx#L116

Reply:
Since `AppBridge` is deprecated and was supposed to be removed already (Dan & Tim we're aiming for last November iirc) we really shouldn't be added onto the API. If this is just a type change can you hack it in your app with casting?

Reply:
I think it needs to be a part of AppBridge first as @vividviolet said. I will close this PR when I can figure out how to do that 😂